runtime/doc/api.txt

```diff
@@ -624,8 +625,7 @@ nvim_create_buf({listed}, {scratch})                       *nvim_create_buf()*
                 Return: ~
                     Buffer handle, or 0 on error
 
-                                                             *nvim_open_win()*
-nvim_open_win({buffer}, {enter}, {width}, {height}, {options})
+nvim_open_win({buffer}, {enter}, {options})                  *nvim_open_win()*
```

Comment:
`{options}` parameter should be renamed to `{config}`. 

Probably we should never say "options" anywhere if they don't set actual vim options. However `{opt}` (not `{options}`)  is ok on API functions with optional parameters. But in this particular case `nvim_win_get_config` explicitly refers to this parameter on `nvim_open_win`, so `{config}` seems appropriate.